lang/pure/Portfile

```diff
@@ -28,18 +28,18 @@ long_description        Pure is a functional programming language based on \
                         are licensed under LGPL-3+, the interpreter is GPL-3+, \
                         and the examples are BSD-licensed.
 
-bitbucket.tarball_from  downloads
+github.tarball_from     releases
 distname                ${name}-${version}
 
-checksums               rmd160  a3ad0dccdb71eab3aa8ee15d3a04f5e3111208a1 \
-                        sha256  42df6832476e8bee3a7ca179671284c1edd7bc82b71062fa0de62fd2117ee676
+checksums               rmd160  68f52aafe4b378fbafe7bb43fdee5612f1acd666 \
+                        sha256  442de3ccc98f55e7c04a7470771bd8d92e49a43f897964cc5b191c6b7a2b9dc2
```

Comment:
Could you add the `size` parameter (the filesize in bytes) to the `checksums` here and in the other port?

Reply:
Done.